python/ray/ray_constants.py

```diff
@@ -215,3 +215,5 @@ def to_memory_units(memory_bytes, round_up):
 # Max 64 bit integer value, which is needed to ensure against overflow
 # in C++ when passing integer values cross-language.
 MAX_INT64_VALUE = 9223372036854775807
+
+NODE_RESOURCES_TURNING_EMPTY = "_TURNING_EMPTY_"
```

Comment:
Can you add a comment about what this is?

Reply:
Forget this please. I swithed to a new bool flag from this specified key to indicate if resources has changed. This constants has been removed in newest commit. :)